TypeCobol.Test/Report/ReportTestHelper.cs

```diff
@@ -73,7 +73,7 @@ public static ReturnCode ParseWithNodeListenerReportCompare(string fileName, str
                 parser.Init(input, typeCobolOption, format, new List<string>() { copyFolder });
                 parser.Parse(input);
 
-                var allDiags = parser.Results.AllDiagnostics();
+                var allDiags = parser.Results.AllDiagnostics().Where(d => d.Info.Severity != TypeCobol.Compiler.Diagnostics.Severity.Warning).ToList();
```

Comment:
Same as TestLTNV, add a comment to explain skipping warnings.

Reply:
fixed; new commit
https://github.com/TypeCobolTeam/TypeCobol/tree/68d3be7fd420c735f36d49da44e7f495213c4bc0